test/stdlib/String.test.js

```diff
@@ -84,4 +84,76 @@ describe("global string functions", () => {
             ).toEqual(new BrsString(""));
         });
     });
-});
\ No newline at end of file
+
+    describe("Right", () => {
+        it("get last n characters in a string longer than n characters", () => {
+            expect(
+                Right.call(interpreter, new BrsString("pineapple"), new Int32(4))
+            ).toEqual(new BrsString("pple"));
+        });
+
+        it("get the original string back with n larger than string length", () => {
+            expect(
+                Right.call(interpreter, new BrsString("boy"), new Int32(5))
+            ).toEqual(new BrsString("boy"));
+        });
+
+        it("get back empty string when character length is 0 or negative", () => {
+            expect(
+                Right.call(interpreter, new BrsString("apple"), new Int32(0))
+            ).toEqual(new BrsString(""));
+
+            expect(
+                Right.call(interpreter, new BrsString("apple"), new Int32(-5))
+            ).toEqual(new BrsString(""));
+        });
+    });
+
+    describe("Instr", () => {
+        it("returns 0 if the string is not found", () => {
+            expect(
+                Instr.call(interpreter, new Int32(1), new BrsString("apple"), new BrsString("orange"))
+            ).toEqual(new Int32(0));
+        });
+
+        it("returns the index of the first found string", () => {
+            expect(
+                Instr.call(interpreter, new Int32(1), new BrsString("apple"), new BrsString("a"))
+            ).toEqual(new Int32(1));
+        });
+
+        it("returns the index of the first found after the starting index passed in", () => {
```

Comment:
I *love* that the `it` description reads like a full sentence describing the behavior here.  Consider using that for the other test cases you added here &mdash; these are super easy to read!

Reply:
I agree about making tests easy to read. I will work on the others. (excuse: It was late and my eyes were a bit blurry 😄)